src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -42,23 +53,23 @@ private ReadFilterLibrary(){ /*no instance*/ }
 
     public static class MappedReadFilter extends ReadFilter {
         private static final long serialVersionUID = 1L;
-        @Override public boolean test(final GATKRead read){
+        @Override public boolean test(final GATKRead read) {
             return !read.isUnmapped();}}
 
     public static class MappingQualityAvailableReadFilter extends ReadFilter {
         private static final long serialVersionUID = 1L;
-        @Override public boolean test(final GATKRead read){
+        @Override public boolean test(final GATKRead read) {
             return read.getMappingQuality() != QualityUtils.MAPPING_QUALITY_UNAVAILABLE;}}
 
     public static class MappingQualityNotZeroReadFilter extends ReadFilter {
         private static final long serialVersionUID = 1L;
-        @Override public boolean test(final GATKRead read){
+        @Override public boolean test(final GATKRead read) {
             return read.getMappingQuality() != 0;}}
 
     /**
      * Reads that either have a mate that maps to the same contig, or don't have a mapped mate.
      */
-    public static class MateOnSameContigOrNoMappedMateReadFilter extends ReadFilter {
```

Comment:
I think the original name is clearer.  It encapsulates the fact that it accepts reads with either no mate, or reads with an unmapped mate.  It's unclear what the new name does with unpaired reads.